django/db/backends/mysql/features.py

```diff
@@ -116,6 +114,19 @@ def needs_explain_extended(self):
         # EXTENDED is deprecated (and not required) in MySQL 5.7.
         return not self.connection.mysql_is_mariadb and self.connection.mysql_version < (5, 7)
 
+    @cached_property
+    def needs_analyze_statement(self):
+        # EXPLAIN ANALYZE was replaced by ANALYZE in MariaDB 10.1.0.
+        return self.connection.mysql_is_mariadb
+
+    @cached_property
+    def supported_explain_formats(self):
+        # Alias MySQL's TRADITIONAL to TEXT for consistency with other backends.
+        formats = {'JSON', 'TEXT', 'TRADITIONAL'}
+        if not self.connection.mysql_is_mariadb and self.connection.mysql_version >= (8, 0, 16):
```

Comment:
Maybe we could split this into a property itself? `supports_tree_explain_format`?

Reply:
I think that would introduce too many additional properties, and we can use a simple containment check on the set anyway.